concent_api/core/tests/utils.py

```diff
@@ -263,6 +264,11 @@ def _test_response(self, response, status, key, message_type = None, fields = No
         else:
             self.assertEqual(len(response.content), 0)
 
+        if nested_message_verifiable_by is not None:
+            for nested_message, public_key in nested_message_verifiable_by.items():
```

Comment:
You can add assert here that nested_message_verifiable_by is dict.

Reply:
done